File: Web/Divyanshu-Vareshney_divyanshuvarshney_2125it1075_2/GettingStarted/Digits Of A Number.cpp
#include <iostream>
using namespace std;

int main(int argc, char **argv){
    int n;
    cin >> n;

    //write your code here
    int i,j,digit=0;
    while(n>0){
        i=n%10;
        digit=digit*10+i;
        n=n/10;
    }
    while(digit>0){
        j=digit%10;
        cout<<j<<"\n";
        digit=digit/10;
    }
    //DONE
}

File: Android/sukritoberoi_SukritKaurOberoi_2125cse1065_2/GettingStarted/DigitsOfANumber.java
import java.util.*;
import java.lang.Math;

public class Main {

  public static void main(String[] args) {
    // write your code here
    Scanner sc = new Scanner(System.in);
    long n = sc.nextLong();
    long d = 0, n1 = 0;
    while (n != 0)
    {
      n1 = (n1 * 10) + (n % 10);
      n = n / 10;
      d++;
    }
    while (d != 0)
    {
      System.out.println(n1 % 10);
      n1 = n1 / 10;
      d--;
    }
  }
}